src/main/java/com/kenticocloud/delivery/DeliveryParameterBuilder.java

```diff
@@ -179,6 +180,13 @@ public DeliveryParameterBuilder language(String language) {
         return this;
     }
 
+    public DeliveryParameterBuilder language(Locale language) {
+        if (language != null) {
+            nameValuePairs.add(new BasicNameValuePair(LANGUAGE, language.toString()));
```

Comment:
This might not work.  Although you can manually make a locale in the "<language>-<variant>" syntax, you are actually only setting the language doing so.  Constructing language with a variant with a Java locale is done via one of the following examples:

```
Locale.UK
new Locale("en", "GB")
```

The toString() method on Locale will return a locale formatted as "<language>_<variant>", which doesn't match the spec from the Kentico API.  Unless that format is supported, this method will need to do the proper string formatting.

Reply:
@aweigold so `new Locale("en", "GB")` must return 'en-GB' ?

Reply:
That can work.  You will also want to unit test an edge case where no variant is set, so if just new Locale("en") is passed, you are not sending in "en-".

Reply:
@aweigold if the client passes only `new Locale("en")` is must return only "en"?

Reply:
Correct

Reply:
So it's working correctly

Reply:
@cunhazera "So it's working correctly" as in the Kentico API will accept the underscores?  Or it's working correctly if you just replace the "_" for "-" ?  If the latter, that replacement will work, I was just advising that it would make sense to do a unit test for that case as well.